api/app.js

```diff
@@ -157,9 +155,7 @@ module.exports = function(settings, callback) {
     var errorResponse = {};
     if (err) {
       errorResponse = {
-        'success': false,
         'error': err,
-        'result': null
```

Comment:
what is the reason why we have 'success' and 'result' defined in error message previously?

Reply:
@qibobo  may be a question for you...

Reply:
@boyang9527 I did not change the result content of policy when I did the public api relevant PRs. It should be created in the origin commits of apiserver.